sdk/modelsrepository/Azure.Iot.ModelsRepository/README.md

```diff
@@ -0,0 +1,89 @@
+
+Install the Azure IoT Models Repository client library for .NET with [NuGet][nuget]:
+
+```PowerShell
+Install-Package Azure.Iot.ModelsRepository
+```
+
+View the package details at [nuget.org][modelsrepository_nuget].
+
+### Prerequisites
+
+- A models repository following [Azure IoT conventions][modelsrepository_conventions]
+  - The models repository can be located on the local filesystem or hosted on a webserver.
+  - Azure IoT hosts the global [Azure IoT Models Repository](modelsrepository_iot_endpoint) which the client will point to by default if no Uri is provided.
+
+### Authenticate the Client
+
+Currently no authentication mechanisms are supported in the client. The global endpoint is not tied to an Azure subscription and does not support auth. All models published are meant for anonymous public consumption.
+
+## Key concepts
+
+The Azure IoT Models Repository enables builders to manage and share digital twin models. The models are JSON-LD documents defined using the Digital Twins Definition Language (DTDL).
+
+The repository defines a pattern to store DTDL interfaces in a directory structure based on the Digital Twin Model Identifier (DTMI). You can locate an interface in the repository by converting the DTMI to a relative path. For example, the DTMI "dtmi:com:example:Thermostat;1" translates to /dtmi/com/example/thermostat-1.json.
+
+### Thread safety
+
+We guarantee that all client instance methods are thread-safe and independent of each other ([guideline](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-service-methods-thread-safety)). This ensures that the recommendation of reusing client instances is always safe, even across threads.
+
+### Additional concepts
+<!-- CLIENT COMMON BAR -->
+[Client options](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/README.md#configuring-service-clients-using-clientoptions) |
+[Accessing the response](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/README.md#accessing-http-response-details-using-responset) |
+[Long-running operations](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/README.md#consuming-long-running-operations-using-operationt) |
+[Handling failures](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/README.md#reporting-errors-requestfailedexception) |
+[Diagnostics](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/samples/Diagnostics.md) |
+[Mocking](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/README.md#mocking) |
+[Client lifetime](https://devblogs.microsoft.com/azure-sdk/lifetime-management-and-thread-safety-guarantees-of-azure-sdk-net-clients/)
+<!-- CLIENT COMMON BAR -->
+
+## Examples
+
+You can familiarize yourself with the client using [samples for IoT Models Repository](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/modelsrepository/Azure.Iot.ModelsRepository/samples/).
```

Comment:
Do we want all links aliased?

Reply:
agreed, let's keep all links aliased and consistent

Reply:
Aliased all links except in one sentence "For details, visit <https://cla.microsoft.com.>" which is the same as the DigitalTwins.Core [readme](https://raw.githubusercontent.com/Azure/azure-sdk-for-net/master/sdk/digitaltwins/Azure.DigitalTwins.Core/README.md)